firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigRealtimeHttpClient.java

```diff
@@ -346,6 +348,11 @@ public synchronized void beginRealtimeHttpStream() {
       }
     } catch (IOException e) {
       Log.d(TAG, "Exception connecting to realtime stream. Retrying the connection...");
+      propagateErrors(
+          new FirebaseRemoteConfigServerException(
```

Comment:
Looks like `getResponseCode` always return something reasonable, so we can include that here, too https://docs.oracle.com/javase/7/docs/api/java/net/HttpURLConnection.html#getResponseCode()

Reply:
This is for the case where getResponseCode throws an exceptions and isn't able to return a status code. If it has a non-retryable response code anyways it would be caught in the exceptions below anyways.